### PR TITLE
Improve description for tf.test.is_gpu_available

### PR DESCRIPTION
**Pull request by lc0**
As a part of TensorFlow docs sprints from Munich.

A small explanation for added warning - https://colab.research.google.com/drive/10pT_SQBAbZm9yVcoaACgrAlQ3YbpM_RI

cc @dynamicwebpaige

### CODE REVIEW

**Comment by lc0** [comment]
any updates on this one? @annarev @ymodak @dynamicwebpaige

**Review by dynamicwebpaige — APPROVED** [review]
LGTM! Thanks for adding this description. :slightly_smiling_face:

File: tensorflow/python/framework/test_util.py [hunk 1]
```diff
@@ -1331,13 +1331,17 @@ def decorated(self, *args, **kwargs):
 def is_gpu_available(cuda_only=False, min_cuda_compute_capability=None):
   """Returns whether TensorFlow can access a GPU.
 
+  Warning: if not GPU version of the package is installed, the function would
```
Comment (annarev) [inline tensorflow/python/framework/test_util.py]: "non-GPU" sounds better than "not GPU" to me.

File: tensorflow/python/framework/test_util.py [hunk 2]
```diff
@@ -1331,13 +1331,17 @@ def decorated(self, *args, **kwargs):
...
+  also, return False. Use `tf.test.is_built_with_cuda` to validate if TensorFlow
```
Comment (annarev) [inline tensorflow/python/framework/test_util.py]: Redundant comma here "also, return False"

**Review by annarev — COMMENTED** [review]
Thank you for the change!

**Comment by annarev** [comment]
oops, didn't realize it is already merged.

**Pull request merged**